src/e2e-app/autocomplete/autocomplete-e2e.html

```diff
@@ -0,0 +1,11 @@
+<section>
```

Comment:
One thing that's not clear in the code by itself is that we're trying to move away from standalone e2e test pages. Instead, new e2e tests are based on the examples we publish to material.angular.io. Check out #5735 for an example of adding e2e tests for cards based on the published examples.